packages/mongodb-chatbot-ui/src/ModalView.tsx

```diff
@@ -62,18 +62,20 @@ export function ModalView(props: ModalViewProps) {
   const chatWndowInputBarId = inputBarId ?? "chatbot-modal-input-bar";
 
   return (
-    <Modal
-      className={cx(styles.modal_container({ darkMode }), className)}
-      open={open}
-      size="large"
-      initialFocus={`#${chatWndowInputBarId}`}
-      shouldClose={shouldClose}
-    >
-      <Suspense fallback={null}>
-        {open ? (
+    <Suspense fallback={null}>
+      {open ? (
+        <Modal
+          className={cx(styles.modal_container({ darkMode }), className)}
+          open={open}
+          size="large"
+          initialFocus={
+            !conversation.error ? `#${chatWndowInputBarId}` : undefined
```

Comment:
While you're in here, could you fix this in the various places?
```suggestion
            !conversation.error ? `#${chatWindowInputBarId}` : undefined
```